packages/playground/src/App.tsx

```diff
@@ -197,7 +197,7 @@ function App() {
 export default function AppWrapper() {
   return (
     <PlaygroundQueryClient>
-      <StudioConfigProvider>
+      <StudioConfigProvider endpoint={`http://localhost:${window.MASTRA_SERVER_PORT || 4111}`}>
```

Comment:
What happens when you set a custom `host` in the server config (so host + port)?